domain/utxoindex/store.go

```diff
@@ -0,0 +1,257 @@
+package utxoindex
+
+import (
+	"github.com/golang/protobuf/proto"
+	"github.com/kaspanet/kaspad/domain/consensus/database/serialization"
+	"github.com/kaspanet/kaspad/domain/consensus/model/externalapi"
+	"github.com/kaspanet/kaspad/infrastructure/db/database"
+	"github.com/kaspanet/kaspad/infrastructure/logger"
+	"github.com/pkg/errors"
+)
+
+var utxoIndexBucket = database.MakeBucket([]byte("utxo-index"))
+
+type utxoIndexStore struct {
+	database database.Database
```

Comment:
DBManager was created as a workaround, so that we won't hold references to non-model objects in the model package.
There's no reason to use it outside of those cases.